addons/website_sale/static/src/js/website_sale.editor.js

```diff
@@ -53,11 +53,12 @@ odoo.define('website_sale.editor', function (require) {
 'use strict';
 
 require('web.dom_ready');
```

Comment:
You cannot define options in a module which waits for dom ready. I think you can remove dom_ready but you have to adapt the line 63 (which checks the DOM) and change it into an animation (sAnimatons.registry...) like you did before.